viewbuilder-rx2/src/main/java/com/squareup/viewbuilder/MainAndModalScreen.kt

```diff
@@ -15,6 +15,19 @@
  */
 package com.squareup.viewbuilder
 
+/**
+ * For flows that navigate through a series of main / body screens, sometimes covered by
+ * the occasional modal dialog.
+ */
+typealias StackedMainAndModalScreen<M, D> = MainAndModalScreen<StackScreen<M>, D>
+
+/**
+ * For flows that show a [main] screen, optionally covered by a number of nested
+ * [modals].
+ *
+ * @param M the type of the [main] / body screen, typically `StackScreen<*>`
```

Comment:
This "typically" comment is a bit redundant with the `typealias`. I'm pretty sure the `typealias` carries its weight, I expect it'll be the most common case.